log/gps/gps_destinator_misc.ksy

```diff
@@ -0,0 +1,95 @@
+meta:
+  id: gps_destinator
+  title: Destinator mixed stuff format
+  application: Destinator
+  file-extension:
+    - rte
+    - dat
+  license: GPL-2.0-or-later
+  encoding: utf-16le
+  endian: le
+
+doc: |
+  Destinator formats
+
+doc-ref:
+  - https://web.archive.org/web/20160310115940/http://mozoft.com/d3log.html
+  - https://web.archive.org/web/20071011022251/http://www.destinatortechnologies.net/uk/products/products/bundles/
+  - https://www.gpsbabel.org/htmldoc-development/fmt_destinator_itn.html
+  - https://www.gpsbabel.org/htmldoc-development/fmt_destinator_poi.html
+  - https://github.com/gpsbabel/gpsbabel/blob/e1e195beca8e0f24ca0e41c02d1986af13865cd0/destinator.cc#L215
+
+
+seq:
+  - id: records
+    type: record
+    repeat: eos
+
+types:
+  point:
+    seq:
+      - id: lon
+        type: f8
+      - id: lat
+        type: f8
+      - id: lon_repeat
+        type: f8
+        valid:
+          eq:
+            lon
+      - id: lat_repeat
+        type: f8
+        valid:
+          eq:
+            lat
+
+  record:
+    seq: 
+      - id: tag
+        type: strz
+      - id: short_name
+        type: strz
+      - id: notes
+        type: strz
+      - id: payload
+        type:
+          switch-on: tag
+          cases:
+            '"Dynamic POI"': favorite
+            '"City->Street"': itenirary
+
+
+  itenirary:
+    seq:
+      - id: unkn0
```

Comment:
I think "unknown" can be spelled out, it's just 3 extra letters 🙂

(same for more "unkn" fields below)